internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/server/VeniceServerGrpcTest.java

```diff
@@ -0,0 +1,87 @@
+package com.linkedin.venice.server;
+
+import com.linkedin.venice.listener.grpc.VeniceReadServiceClient;
+import com.linkedin.venice.listener.grpc.VeniceReadServiceServer;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class VeniceServerGrpcTest {
+  private VeniceReadServiceServer server;
+  private VeniceReadServiceClient client;
+
+  private void createServer(int port) {
+    this.server = new VeniceReadServiceServer(port);
+    this.server.start();
+  }
+
+  private void createClient(String addr, int port) {
+    this.client = new VeniceReadServiceClient(addr, port);
+  }
+
+  private void stopServer() {
```

Comment:
IMO `stopServer()` and `shutdownClient()` methods here are not needed. We can directly call stop and shutdown.